.travis.yml

```diff
@@ -0,0 +1,29 @@
+# This config file for Travis CI
+sudo: required
+dist: trusty
+services:
+  - docker
+language: cpp
+cache: ccache
+compiler: gcc
```

Comment:
Do you have a specific reason for specifying this? The default should work afaik.

Reply:
the compiler is not used at all in `industrial_ci`.